LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/intrinsic-sizing/masonry-intrinsic-sizing-cols-001-auto.html

```diff
@@ -48,7 +48,7 @@
   <item>2</item>
   <item>3</item>
   <item>4</item>
-  <item style="width:2ch; grid-column:2">5</item>
```

Comment:
Was this accidental? Seems to be inconsistent from the rest of the tests

Reply:
This was accidental.